sabnzbd/newsunpack.py

```diff
@@ -1053,6 +1053,10 @@ def seven_extract_core(sevenset, extensions, extraction_path, one_folder, delete
         # We can output a more general error
         ret = 1
         msg = T('ERROR: CRC failed in "%s"') % setname_from_path(sevenset)
+    if ret == 2 and (("Disk full." in output) or ("No space left on device" in output)):
+        # note: the above does not work with 7z version 16.02, and does from with 7z 19.00 and higher
+        ret = 1  # to avoid ret = 2, causing error about password
+        msg = T('ERROR: Not enough disk space for unzipping "%s"') % setname_from_path(sevenset)
```

Comment:
No no, we don't do that `ERROR` stuff anymore. That's from ancient times.
Let's make it:
`Unpacking failed, no space left on device.

Reply:
plain "unpacking"? Not "unzipping"?

Reply:
> > note to self, and @Safihre : after the failed download, SAB says "complete". That is not good, I assume?
> 
> No that's wrong, it should set the right flags to make it show failed.

Found the cause:

- with just a .7z file, it results in nice failure message in the GUI, including the download itself
- however, with several .7z.001 etc files, there is a failure message in the GUI, but the GUI says "completed" for the download. Probable cause: SAB/7z does a two-pass method on .7z.001 files: first converts it to one big .7z (OK), and then unpacks that (not OK if too little disk space)

See below.

And proof no translation is needed: it works in Nederlands:

![image](https://user-images.githubusercontent.com/1273502/147383397-8bfa1ceb-8d9e-4d2b-8506-5e0fdce1d447.png)